transcend/data_source_data_silos.go

```diff
@@ -41,7 +41,7 @@ func dataSourceDataSilosRead(ctx context.Context, d *schema.ResourceData, m inte
 	var diags diag.Diagnostics
 
 	var query struct {
-		DataSilos types.DataSilosPayload `graphql:"dataSilos(filterBy: $filterByInput)"`
+		DataSilos types.DataSilosPayload `graphql:"dataSilos(filterBy: $filterByInput, first: 100)"`
```

Comment:
What's the rationale here? And do we wanna make this configurable?

Reply:
Right now only 10 data silos can be returned (which is the default from our backend), but a customer had 11 data silos. I figured 100 was bigger than 10 🤷